docs/src/pages/[platform]/connected-components/liveness/faqs.mdx

```diff
@@ -0,0 +1,25 @@
+import { Fragment } from '@/components/Fragment';
+
+## Frequently Asked Questions
+
+The following issues are presented with corresponding troubleshooting steps. Please check them out in case you run into issues like these -- if the remedies don't work, please file an issue in our GitHub repositories.
+
+### Error: Cognito user pool does not allow access to unauthenticated identities
+
+First, make sure you have followed the [prerequisites for setting up Amplify resources](#step-1-configure-auth). Second, you can also go into the Cognito identity pool configured in your Amplify app, edit it and select the checkbox labeled "Enable access to unauthenticated identities" -- then save the changes. Finally, if using React, ensure that your `aws-exports.js` file exists and is being imported correctly into your app.
+
+### Error: Server error - Session not found 
+
+Ensure that the region you are using to create the liveness session using your backend is the same as the one used within your app to start it. Also ensure that the session ID returned from the backend when creating the session is the same as the one started by the app. Note that whether the liveness check succeeds or fails, a new session is required.
+
+### Error: AccessDeniedError when calling Rekognition APIs (CreateFaceLivenessSession, etc.)
```

Comment:
@tjleing Can you confirm that this is the exact error message you got? We want to make sure this is searchable

Reply:
error text: "An error occurred (AccessDeniedException) when calling the CreateFaceLivenessSession operation:" (with nothing after the colon).  is `AccessDeniedException` enough to communicate this?

Reply:
The concern isn't understanding, What I want is for someone to be able to google "An error occurred (AccessDeniedException) when calling the CreateFaceLivenessSession". Can you put the full error message in a code block below just so it's searchable?

Reply:
can do!